### ISSUE TITLE
binance watch_orders not returning updates

### ISSUE THREAD

**Issue by lookis** [ccxt/ccxt#6641]
MUST READ THIS BEFORE SUBMITTING ISSUES (read the link, then delete this message before submitting):

https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#how-to-submit-an-issue

Please make sure your local version of ccxt is up to date with this repo. You can check by comparing the output of `ccxt.version` to https://github.com/ccxt/ccxt/blob/master/package.json#L3

- OS: macOS 10.15.3
- Programming Language version: python 3.7.6
- CCXT version: pro 0.0.79
- Exchange: binance
- Method: watch_orders

It seems watch_orders never got data, I have tried watch_balance and balance notified correctly after place an order and cancel it, but watch_orders never return neither NEW or CANCELED or REJECTED

**Comment by kroitor** [comment]
Will get this resolved shortly, thx for reporting!

**Comment by frosty00** [comment]
Hi, 

this is actually the intended behaviour,

have you tried running the script so it is blocked and at the same time logging into binance and creating an order?

if you do this you will find the order data is streamed to you (including partial fills)

thanks for using ccxt pro, let us know if you still have any troubles

**Comment by lookis** [comment]
```python
# after initializing binance with apikey and secret
binance = ...
async def orders():
    orders = await binance.watch_orders()
    print(orders)

asyncio.get_event_loop().run_until_complete(orders())
```

I have got the log with verbose = True,

but still no return 
```
2020-03-18T07:13:04.830Z message {"e":"executionReport","E":1584515584720,"s":"BTCUSDT","c":"web_9b48108da354478dba0b0067ee3ddefb","S":"BUY","o":"LIMIT","f":"GTC","q":"0.10000000","p":"4000.00000000","P":"0.00000000","F":"0.00000000","g":-1,"C":"","x":"NEW","X":"NEW","r":"NONE","i":1527713885,"l":"0.00000000","z":"0.00000000","L":"0.00000000","n":"0","N":null,"T":1584515584718,"t":-1,"I":3312492470,"w":true,"m":false,"M":false,"O":1584515584718,"Z":"0.00000000","Y":"0.00000000","Q":"0.00000000"}
2020-03-18T07:13:04.919Z message {"e":"outboundAccountInfo","E":1584515584720,"m":10,"t":10,"b":0,"s":0,"T":true,"W":true,"D":true,"u":1584515584718,"B":[{"a":"BTC","f":"0.00000000","l":"0.00000000"},{"a":"LTC","f":"0.00000000","l":"0.00000000"},{"a":"ETH","f":"0.00000000","l":"0.00000000"},{"a":"NEO","f":"0.00000000","l":"0.00000000"},{"a":"BNB","f":"0.00000000","l":"0.00000000"},{"a":"QTUM","f":"0.00000000","l":"0.00000000"},{"a":"EOS","f":"0.00000000","l":"0.00000000"},{"a":"SNT","f":"0.00000000","l":"0.00000000"},{"a":"BNT","f":"0.00000000","l":"0.00000000"},{"a":"GAS","f":"0.00000000","l":"0.00000000"},{"a":"BCC","f":"0.00000000","l":"0.00000000"},{"a":"USDT","f":"0.01396062","l":"400.00000000"},{"a":"HSR","f":"0.00000000","l":"0.00000000"},{"a":"OAX","f":"0.00000000","l":"0.00000000"},{"a":"DNT","f":"0.00000000","l":"0.00000000"},{"a":"MCO","f":"0.00000000","l":"0.00000000"},{"a":"ICN","f":"0.00000000","l":"0.00000000"},{"a":"ZRX","f":"0.00000000","l":"0.00000000"},{"a":"OMG","f":"0.00000000","l":"0.00000000"},{"a":"WTC","f":"0.00000000","l":"0.00000000"},{"a":"YOYO","f":"0.00000000","l":"0.00000000"},{"a":"LRC","f":"0.00000000","l":"0.00000000"},{"a":"TRX","f":"0.00000000","l":"0.00000000"},{"a":"SNGLS","f":"0.00000000","l":"0.00000000"},{"a":"STRAT","f":"0.00000000","l":"0.00000000"},{"a":"BQX","f":"0.00000000","l":"0.00000000"},{"a":"FUN","f":"0.00000000","l":"0.00000000"},{"a":"KNC","f":"0.00000000","l":"0.00000000"},{"a":"CDT","f":"0.00000000","l":"0.00000000"},{"a":"XVG","f":"0.00000000","l":"0.00000000"},{"a":"IOTA","f":"0.00000000","l":"0.00000000"},{"a":"SNM","f":"0.00000000","l":"0.00000000"},{"a":"LINK","f":"0.00000000","l":"0.00000000"},{"a":"CVC","f":"0.00000000","l":"0.00000000"},{"a":"TNT","f":"0.00000000","l":"0.00000000"},{"a":"REP","f":"0.00000000","l":"0.00000000"},{"a":"MDA","f":"0.00000000","l":"0.00000000"},{"a":"MTL","f":"0.00000000","l":"0.00000000"},{"a":"SALT","f":"0.00000000","l":"0.00000000"},{"a":"NULS","f":"0.00000000","l":"0.00000000"},{"a":"SUB","f":"0.00000000","l":"0.00000000"},{"a":"STX","f":"0.00000000","l":"0.00000000"},{"a":"MTH","f":"0.00000000","l":"0.00000000"},{"a":"ADX","f":"0.00000000","l":"0.00000000"},{"a":"ETC","f":"0.00000000","l":"0.00000000"},{"a":"ENG","f":"0.00000000","l":"0.00000000"},{"a":"ZEC","f":"0.00000000","l":"0.00000000"},{"a":"AST","f":"0.00000000","l":"0.00000000"},{"a":"GNT","f":"0.00000000","l":"0.00000000"},{"a":"DGD","f":"0.00000000","l":"0.00000000"},{"a":"BAT","f":"0.00000000","l":"0.00000000"},{"a":"DASH","f":"0.00000000","l":"0.00000000"},{"a":"POWR","f":"0.00000000","l":"0.00000000"},{"a":"BTG","f":"0.00000000","l":"0.00000000"},{"a":"REQ","f":"0.00000000","l":"0.00000000"},{"a":"XMR","f":"0.00000000","l":"0.00000000"},{"a":"EVX","f":"0.00000000","l":"0.00000000"},{"a":"VIB","f":"0.00000000","l":"0.00000000"},{"a":"ENJ","f":"0.00000000","l":"0.00000000"},{"a":"VEN","f":"0.00000000","l":"0.00000000"},{"a":"ARK","f":"0.00000000","l":"0.00000000"},{"a":"XRP","f":"0.00000000","l":"0.00000000"},{"a":"MOD","f":"0.00000000","l":"0.00000000"},{"a":"STORJ","f":"0.00000000","l":"0.00000000"},{"a":"KMD","f":"0.00000000","l":"0.00000000"},{"a":"RCN","f":"0.00000000","l":"0.00000000"},{"a":"EDO","f":"0.00000000","l":"0.00000000"},{"a":"DATA","f":"0.00000000","l":"0.00000000"},{"a":"DLT","f":"0.00000000","l":"0.00000000"},{"a":"MANA","f":"0.00000000","l":"0.00000000"},{"a":"PPT","f":"0.00000000","l":"0.00000000"},{"a":"RDN","f":"0.00000000","l":"0.00000000"},{"a":"GXS","f":"0.00000000","l":"0.00000000"},{"a":"AMB","f":"0.00000000","l":"0.00000000"},{"a":"ARN","f":"0.00000000","l":"0.00000000"},{"a":"BCPT","f":"0.00000000","l":"0.00000000"},{"a":"CND","f":"0.00000000","l":"0.00000000"},{"a":"GVT","f":"0.00000000","l":"0.00000000"},{"a":"POE","f":"0.00000000","l":"0.00000000"},{"a":"BTS","f":"0.00000000","l":"0.00000000"},{"a":"FUEL","f":"0.00000000","l":"0.00000000"},{"a":"XZC","f":"0.00000000","l":"0.00000000"},{"a":"QSP","f":"0.00000000","l":"0.00000000"},{"a":"LSK","f":"0.00000000","l":"0.00000000"},{"a":"BCD","f":"0.00000000","l":"0.00000000"},{"a":"TNB","f":"0.00000000","l":"0.00000000"},{"a":"ADA","f":"0.00000000","l":"0.00000000"},{"a":"LEND","f":"0.00000000","l":"0.00000000"},{"a":"XLM","f":"0.00000000","l":"0.00000000"},{"a":"CMT","f":"0.00000000","l":"0.00000000"},{"a":"WAVES","f":"0.00000000","l":"0.00000000"},{"a":"WABI","f":"0.00000000","l":"0.00000000"},{"a":"GTO","f":"0.00000000","l":"0.00000000"},{"a":"ICX","f":"0.00000000","l":"0.00000000"},{"a":"OST","f":"0.00000000","l":"0.00000000"},{"a":"ELF","f":"0.00000000","l":"0.00000000"},{"a":"AION","f":"0.00000000","l":"0.00000000"},{"a":"WINGS","f":"0.00000000","l":"0.00000000"},{"a":"BRD","f":"0.00000000","l":"0.00000000"},{"a":"NEBL","f":"0.00000000","l":"0.00000000"},{"a":"NAV","f":"0.00000000","l":"0.00000000"},{"a":"VIBE","f":"0.00000000","l":"0.00000000"},{"a":"LUN","f":"0.00000000","l":"0.00000000"},{"a":"TRIG","f":"0.00000000","l":"0.00000000"},{"a":"APPC","f":"0.00000000","l":"0.00000000"},{"a":"CHAT","f":"0.00000000","l":"0.00000000"},{"a":"RLC","f":"0.00000000","l":"0.00000000"},{"a":"INS","f":"0.00000000","l":"0.00000000"},{"a":"PIVX","f":"0.00000000","l":"0.00000000"},{"a":"IOST","f":"0.00000000","l":"0.00000000"},{"a":"STEEM","f":"0.00000000","l":"0.00000000"},{"a":"NANO","f":"0.00000000","l":"0.00000000"},{"a":"AE","f":"0.00000000","l":"0.00000000"},{"a":"VIA","f":"0.00000000","l":"0.00000000"},{"a":"BLZ","f":"0.00000000","l":"0.00000000"},{"a":"SYS","f":"0.00000000","l":"0.00000000"},{"a":"RPX","f":"0.00000000","l":"0.00000000"},{"a":"NCASH","f":"0.00000000","l":"0.00000000"},{"a":"POA","f":"0.00000000","l":"0.00000000"},{"a":"ONT","f":"0.00000000","l":"0.00000000"},{"a":"ZIL","f":"0.00000000","l":"0.00000000"},{"a":"STORM","f":"0.00000000","l":"0.00000000"},{"a":"XEM","f":"0.00000000","l":"0.00000000"},{"a":"WAN","f":"0.00000000","l":"0.00000000"},{"a":"WPR","f":"0.00000000","l":"0.00000000"},{"a":"QLC","f":"0.00000000","l":"0.00000000"},{"a":"GRS","f":"0.00000000","l":"0.00000000"},{"a":"CLOAK","f":"0.00000000","l":"0.00000000"},{"a":"LOOM","f":"0.00000000","l":"0.00000000"},{"a":"BCN","f":"0.00000000","l":"0.00000000"},{"a":"TUSD","f":"0.00000000","l":"0.00000000"},{"a":"ZEN","f":"0.00000000","l":"0.00000000"},{"a":"SKY","f":"0.00000000","l":"0.00000000"},{"a":"THETA","f":"0.00000000","l":"0.00000000"},{"a":"IOTX","f":"0.00000000","l":"0.00000000"},{"a":"QKC","f":"0.00000000","l":"0.00000000"},{"a":"AGI","f":"0.00000000","l":"0.00000000"},{"a":"NXS","f":"0.00000000","l":"0.00000000"},{"a":"SC","f":"0.00000000","l":"0.00000000"},{"a":"NPXS","f":"0.00000000","l":"0.00000000"},{"a":"KEY","f":"0.00000000","l":"0.00000000"},{"a":"NAS","f":"0.00000000","l":"0.00000000"},{"a":"MFT","f":"0.00000000","l":"0.00000000"},{"a":"DENT","f":"0.00000000","l":"0.00000000"},{"a":"ARDR","f":"0.00000000","l":"0.00000000"},{"a":"HOT","f":"0.00000000","l":"0.00000000"},{"a":"VET","f":"0.00000000","l":"0.00000000"},{"a":"DOCK","f":"0.00000000","l":"0.00000000"},{"a":"POLY","f":"0.00000000","l":"0.00000000"},{"a":"ONG","f":"0.00000000","l":"0.00000000"},{"a":"PHX","f":"0.00000000","l":"0.00000000"},{"a":"HC","f":"0.00000000","l":"0.00000000"},{"a":"GO","f":"0.00000000","l":"0.00000000"},{"a":"PAX","f":"0.00000000","l":"0.00000000"},{"a":"RVN","f":"0.00000000","l":"0.00000000"},{"a":"DCR","f":"0.00000000","l":"0.00000000"},{"a":"USDC","f":"0.00000000","l":"0.00000000"},{"a":"MITH","f":"0.00000000","l":"0.00000000"},{"a":"BCHABC","f":"0.00000000","l":"0.00000000"},{"a":"BCHSV","f":"0.00000000","l":"0.00000000"},{"a":"REN","f":"0.00000000","l":"0.00000000"},{"a":"BTT","f":"0.00000000","l":"0.00000000"},{"a":"USDS","f":"0.00000000","l":"0.00000000"},{"a":"FET","f":"0.00000000","l":"0.00000000"},{"a":"TFUEL","f":"0.00000000","l":"0.00000000"},{"a":"CELR","f":"0.00000000","l":"0.00000000"},{"a":"MATIC","f":"0.00000000","l":"0.00000000"},{"a":"ATOM","f":"0.00000000","l":"0.00000000"},{"a":"PHB","f":"0.00000000","l":"0.00000000"},{"a":"ONE","f":"0.00000000","l":"0.00000000"},{"a":"FTM","f":"0.00000000","l":"0.00000000"},{"a":"BTCB","f":"0.00000000","l":"0.00000000"},{"a":"USDSB","f":"0.00000000","l":"0.00000000"},{"a":"CHZ","f":"0.00000000","l":"0.00000000"},{"a":"COS","f":"0.00000000","l":"0.00000000"},{"a":"ALGO","f":"0.00000000","l":"0.00000000"},{"a":"ERD","f":"0.00000000","l":"0.00000000"},{"a":"DOGE","f":"0.00000000","l":"0.00000000"},{"a":"BGBP","f":"0.00000000","l":"0.00000000"},{"a":"DUSK","f":"0.00000000","l":"0.00000000"},{"a":"ANKR","f":"0.00000000","l":"0.00000000"},{"a":"WIN","f":"0.00000000","l":"0.00000000"},{"a":"TUSDB","f":"0.00000000","l":"0.00000000"},{"a":"COCOS","f":"0.00000000","l":"0.00000000"},{"a":"PERL","f":"0.00000000","l":"0.00000000"},{"a":"TOMO","f":"0.00000000","l":"0.00000000"},{"a":"BUSD","f":"0.00000000","l":"0.00000000"},{"a":"BAND","f":"0.00000000","l":"0.00000000"},{"a":"BEAM","f":"0.00000000","l":"0.00000000"},{"a":"HBAR","f":"0.00000000","l":"0.00000000"},{"a":"XTZ","f":"0.00000000","l":"0.00000000"},{"a":"NGN","f":"0.00000000","l":"0.00000000"},{"a":"NKN","f":"0.00000000","l":"0.00000000"},{"a":"EUR","f":"0.00000000","l":"0.00000000"},{"a":"KAVA","f":"0.00000000","l":"0.00000000"},{"a":"RUB","f":"0.00000000","l":"0.00000000"},{"a":"ARPA","f":"0.00000000","l":"0.00000000"},{"a":"TRY","f":"0.00000000","l":"0.00000000"},{"a":"CTXC","f":"0.00000000","l":"0.00000000"},{"a":"BCH","f":"0.00000000","l":"0.00000000"},{"a":"TROY","f":"0.00000000","l":"0.00000000"},{"a":"VITE","f":"0.00000000","l":"0.00000000"},{"a":"FTT","f":"0.00000000","l":"0.00000000"},{"a":"OGN","f":"0.00000000","l":"0.00000000"},{"a":"DREP","f":"0.00000000","l":"0.00000000"},{"a":"BULL","f":"0.00000000","l":"0.00000000"},{"a":"BEAR","f":"0.00000000","l":"0.00000000"},{"a":"ETHBULL","f":"0.00000000","l":"0.00000000"},{"a":"ETHBEAR","f":"0.00000000","l":"0.00000000"},{"a":"XRPBULL","f":"0.00000000","l":"0.00000000"},{"a":"XRPBEAR","f":"0.00000000","l":"0.00000000"},{"a":"EOSBULL","f":"0.00000000","l":"0.00000000"},{"a":"EOSBEAR","f":"0.00000000","l":"0.00000000"},{"a":"TCT","f":"0.00000000","l":"0.00000000"},{"a":"WRX","f":"0.00000000","l":"0.00000000"},{"a":"LTO","f":"0.00000000","l":"0.00000000"},{"a":"MBL","f":"0.00000000","l":"0.00000000"},{"a":"COTI","f":"0.00000000","l":"0.00000000"},{"a":"BNBBULL","f":"0.00000000","l":"0.00000000"},{"a":"BNBBEAR","f":"0.00000000","l":"0.00000000"}]}
2020-03-18T07:13:04.922Z message {"e":"outboundAccountPosition","E":1584515584720,"u":1584515584718,"B":[{"a":"BTC","f":"0.00000000","l":"0.00000000"},{"a":"BNB","f":"0.00000000","l":"0.00000000"},{"a":"USDT","f":"0.01396062","l":"400.00000000"}]}
```

**Comment by frosty00** [comment]
yeah it is because @kroitor has not merged https://github.com/kroitor/ccxt.pro/pull/6

**Comment by kroitor** [comment]
@lookis will fix it very soon, plz hold on, will update you here. Thx again!

**Comment by lookis** [comment]
thx, you both reply so quickly, awesome

**Comment by kroitor** [comment]
@lookis i've merged the PR by @frosty00 that fixes this issue temporarily – the method will return each new order update for now. This is available in CCXT Pro 0.0.082+

We will soon change this behavior to actually return all orders and will notify you again when that happens. Will also document the usage thoroughly. Thank you!
